Library/Formula/binwalk.rb

```diff
@@ -1,41 +1,21 @@
 class Binwalk < Formula
   desc "Searches a binary image for embedded files and executable code"
   homepage "http://binwalk.org/"
-  revision 1
 
   stable do
-    url "https://github.com/devttys0/binwalk/archive/v2.0.1.tar.gz"
-    sha256 "90ee8426d71e91b62dfe4a1446c457bc7835b475b28717859e275a0494403959"
-
-    # Fixes OS-X-specific issues no longer relevant in HEAD:
-    #
-    # * Fixes OS X bug in 'setup.py':
-    #   * See <https://github.com/devttys0/binwalk/commit/8278229cae4c2c354ffc5bfc3bcef0fd1d9bf2b3>
-    #     and <https://github.com/devttys0/binwalk/commit/76ff729a19552e6e58505ca64b1d4ce2325e7ac0>.
-    # * Fixes library lookup for non-standard Homebrew installations:
-    #   * See upstream issue <https://github.com/devttys0/binwalk/issues/130>
-    #     for the details.
-    #   * The fix is Homebrew-specific as it uses HOMEBREW_PREFIX, that is
-    #     implicitly replace with the actual Homebrew prefix, in the patch.
-    patch :DATA
+    url "https://github.com/devttys0/binwalk/archive/v2.1.1.tar.gz"
+    sha256 "1b70a5b03489d29f60fef18008a2164974234874faab48a4f47ec53d461d284a"
```

Comment:
As there's nothing else in this `stable do` block you can just put the `url`/`sha256` below `homepage`.